### PR TITLE
add doc for third party ingress

### PR DESCRIPTION
**Pull request by howardjohn**
How to do this is a common question we get. Add some docs.

depends on https://github.com/istio/istio.io/pull/13115

### CODE REVIEW

File: content/en/docs/ops/integrations/loadbalancers/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: Third Party Load Balancers
+description: How to integrate Istio with third party load balancers.
+weight: 90
+keywords: [traffic-management,ingress]
+owner: istio/wg-networking-maintainers
+test: no
```
Comment (frankbu) [inline content/en/docs/ops/integrations/loadbalancers/index.md:7]:
```suggestion
test: n/a
```

File: content/en/docs/ops/integrations/loadbalancers/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+{{< mermaid >}}
+graph LR
+    cc((Client))
+    tpi(Third Party Ingress)
+    a(Backend)
+    cc-->tpi-->a
+{{< /mermaid >}}
+
+In this mode, things mostly just work.
+Clients in a service mesh do not need to be aware that the backed they are connecting to have a sidecar.
```
Comment (ericvn) [inline content/en/docs/ops/integrations/loadbalancers/index.md:26]:
```suggestion
Clients in a service mesh do not need to be aware that the backed they are connecting to has a sidecar.
```

File: content/en/docs/ops/integrations/loadbalancers/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+However, the ingress will not use mTLS, which may lead to undesirable behavior.
+As a result, most of the configuration for this setup is around enabling mTLS.
+
+In "chained" mode, we use both the third party ingress *and* Istio's own Gateway in parallel.
```
Comment (ericvn) [inline content/en/docs/ops/integrations/loadbalancers/index.md:30]: I'm not sure I like the word `parallel` here, as it means something different in a circuit and are really in `series`. Not sure I have a bite wording, and maybe just leave it off:
```suggestion
In "chained" mode, we use both the third party ingress *and* Istio's own Gateway.
```

File: content/en/docs/ops/integrations/loadbalancers/index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+This can be useful when you want the functionality of both layers.
+In particular, this is useful with managed cloud load balancers, which have features like global addresses and managed certificates
```
Comment (ericvn) [inline content/en/docs/ops/integrations/loadbalancers/index.md:32]:
```suggestion
In particular, this is useful with managed cloud load balancers, which have features like global addresses and managed certificates.
```

File: content/en/docs/ops/integrations/loadbalancers/index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+Standalone mode with mTLS can be achieved by inserting a sidecar into the Pod of the in-cluster load balancer.
+This typically involves two steps beyond standard sidecar injection:
+
+1. Disable inbound traffic redirection.
+   While not required, typically we only want to use the sidecar for *outbound* traffic - inbound connection from clients is already handled by the load balancer itself.
+   This also allows preserving the original client IP address, which would otherwise be lost by the sidecar.
+   This mode can be enabled by inserting the `traffic.sidecar.istio.io/includeInboundPorts: ""` annotation on the load balancer `Pod`s.
+1. Enable Service routing.
+   Istio sidecars can only properly function when requests are sent to Service's, not to specific pod IPs.
```
Comment (ericvn) [inline content/en/docs/ops/integrations/loadbalancers/index.md:69]:
```suggestion
   Istio sidecars can only properly function when requests are sent to Services, not to specific pod IPs.
```

File: content/en/docs/ops/integrations/loadbalancers/index.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    tpi-->ii
+    ii-->a
+{{< /mermaid >}}
+
+## Cloud Load Balancers
+
+Generally, cloud load balancers will work out of the box in standalone mode without mTLS.
+Vendor specific configuration is required to support chained mode or standalone with mTLS.
+
+### Google HTTP(S) Load Balancer
```
Comment (ericvn) [inline content/en/docs/ops/integrations/loadbalancers/index.md:50]: @frankbu may have other thoughts, but I think we try to keep the documents platform neutral so not sure if this section should be here.

File: content/en/docs/ops/integrations/loadbalancers/index.md (hunk 3 above)
Comment (frankbu) [reply content/en/docs/ops/integrations/loadbalancers/index.md:30]: Maybe `in parallel` -> `in sequence` ?

File: content/en/docs/ops/integrations/loadbalancers/index.md (hunk 2 above)
Comment (frankbu) [reply content/en/docs/ops/integrations/loadbalancers/index.md:26]: also backed -> backend
```suggestion
Clients in a service mesh do not need to be aware that the backend they are connecting to has a sidecar.
```

File: content/en/docs/ops/integrations/loadbalancers/index.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+Istio provides both an ingress and service mesh implementation, which can be used together or separately. While these are designed to work together seamlessly, there are times when integrating with a third party ingress is required. This could be for migration purposes, feature requirements, or personal preferences.
+
+## Integration Modes
+
+In "standalone" mode, the third party ingress is directly sending to backends.
+In this case, the backends presumably have Istio sidecars injected;
```
Comment (frankbu) [inline content/en/docs/ops/integrations/loadbalancers/index.md:15]:
```suggestion
In this case, the backends presumably have Istio sidecars injected.
```

File: content/en/docs/ops/integrations/loadbalancers/index.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+{{< /mermaid >}}
+
+## Cloud Load Balancers
+
+Generally, cloud load balancers will work out of the box in standalone mode without mTLS.
+Vendor specific configuration is required to support chained mode or standalone with mTLS.
+
+### Google HTTP(S) Load Balancer
+
+Integration with Google HTTP(S) Load Balancers works out of the box with standalone mode if mTLS is not required.
```
Comment (frankbu) [inline content/en/docs/ops/integrations/loadbalancers/index.md:52]:
```suggestion
Integration with Google HTTP(S) Load Balancers only works out of the box with standalone mode if mTLS is not required as
```

File: content/en/docs/ops/integrations/loadbalancers/index.md (hunk 6 above)
Comment (ericvn) [reply content/en/docs/ops/integrations/loadbalancers/index.md:50]: OK. We would just expect Any other providers who wanted to provide info to submit a PR to add it here. I'm not sure if that warrants a note or not, but I don't really see any notes elsewhere in the doc.

**Review by ericvn — APPROVED** [review]
Thanks.

**Pull request merged**